lib/qualys/asset/evidence.rb

```diff
@@ -0,0 +1,87 @@
+module Qualys::Asset
+  # This class represents each of the ASSET_DATA_REPORT/GLOSSARY/VULN_DETAILS_LIST/
+  # VULN_DETAILS elements in the Qualys WAS XML document.
```

Comment:
WAS?

Is the actual L2 true? seems like it's a weird class name if so.

Reply:
Copy paste ❌